JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -363,5 +363,16 @@ module Microsoft.ApplicationInsights {
             
             return true;
         }
+
+        public static getAbsoluteUrl = (function () {
+            var a;
+
+            return function (url) {
+                if (!a) a = document.createElement('a');
```

Comment:
are there any drawbacks creating dom elements? Also use `Util.document` to minimize the code better. Is there a way to enforce using proper document object